src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -164,6 +164,11 @@ private static string GetPaddedString(List<string> strings, int[] colWidths, int
                     builder.Append(' ');
                 }
 
+                // Replace any newlines with encoded newline (`n)
+                // Note we can't use Environment.Newline because we don't know that the
+                // Command honors that.
+                strings[i] = strings[i].Replace("\n", "`n");
```

Comment:
So thinking about this some more, it may be possible that the string has a carriage return and a linefeed. So in addition to what you're doing here, we should also handle that so:
```suggestion
                strings[i] = strings[i].Replace("\r\n", "`r`n");
                strings[i] = strings[i].Replace("\n", "`n");
```

This makes sure there are no lingering `\r`.

Reply:
Would it be just as good to do a replace for `\r` and another for `\n` and thus catch all of them by default?

Reply:
Yeah that's true. I guess that would catch the off chance that there's a `\r` by itself.

Reply:
I concur. I've changed it to:

```csharp
                // Replace any newlines with encoded newline/linefeed (`n or `r)
                // Note we can't use Environment.Newline because we don't know that the
                // Command honors that.
                strings[i] = strings[i].Replace("\r", "`r");
                strings[i] = strings[i].Replace("\n", "`n");
```

I don't do many PRs, so I'm not sure what I'm supposed to do to push this through other than pushing it to my `master` branch... I think it's automatic?

Reply:
You should create new branch for every new PR.